seminar1/seminar1_LV2/random.ts

```diff
@@ -39,19 +39,22 @@ const dinner: Dinner= {
     ],
     shuffle(array:Member[]) {
         array.sort(() => Math.random() - 0.5);
-        console.log(array);
         return array;
     },
     organize(array:Member[]) {
-        const dinnerMember=new Array();
+        const dinnerMember:string[]=new Array();
 
         this.shuffle(array);
 
-        dinnerMember[0]=array.find((element)=>element.group==='yb');
-        dinnerMember[1]=array.find((element)=>element.group==='ob');
+        const yb: Member|undefined=array.find((element)=>element.group==='yb');
+        const ob: Member|undefined=array.find((element)=>element.group==='ob');
```

Comment:
```suggestion
        const yb: Member | undefined = array.find((element) => element.group==='yb');
        const ob: Member | undefined = array.find((element) => element.group==='ob');
```
Javascript 코딩 컨벤션 같은 것을 참고하시면 가독성 있게 띄어쓰기/인덴팅하는 기준도 나와있으니 이를 참고해보는 것도 좋을 것 같습니다!!

Reply:
조언 정말 감사드립니다 앞으로도 신경 써야겠네요! 수정해서 커밋 했습니다 😆